### DISCUSSION TITLE
Themes do not work over ssh

### DISCUSSION THREAD

**Discussion by AlexNomadrg** [helix-editor/helix#6891] [Q&A]
If I try to set the theme I get `theme requires true color support`. Is there a way to enable it over ssh? 

I tried adding `SendEnv COLORTERM` to `/etc/ssh/ssh_config`  and setting `COLORTERM` to `truecolor`, neither worked.

**Comment by kirawi** [comment]
You can override it manually: https://docs.helix-editor.com/configuration.html#editor-section
```toml
# config.toml
[editor]
true-color = true
```
